test/SlowTests/Bugs/ConnectionStringParsing.cs

```diff
@@ -39,5 +39,26 @@ public void EnsureWellFormedConnectionStrings_Parsing_FailWithUnknownParameter()
             var dbParser = ConnectionStringParser<RavenConnectionStringOptions>.FromConnectionString("memory=true");
             Assert.Throws<ArgumentException>(() => dbParser.Parse());
         }
+
+        [Fact]
+        public void EnsureWellFormedConnectionStrings_ParsingMultiUrls_Successful()
+        {
+            var connectionStringParser = ConnectionStringParser<RavenConnectionStringOptions>.FromConnectionString("Urls=http://localhost:10301,http://localhost:10302,http://localhost:10303;");
```

Comment:
The Format is going to change in this issue https://github.com/ravendb/ravendb/pull/2556